tests/Models/SmimeEncryptionCertificateTest.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace SilverStripe\SmimeForms\Tests;
+namespace SilverStripe\SmimeForms\Tests\Models;
```

Comment:
Organising files into respective namespaced folders rather than all files in one root level